### ISSUE TITLE
deleting files isn't possible from the player tab

### ISSUE THREAD

**Issue by puchu** [SimpleMobileTools/Simple-Voice-Recorder#175]
although i gave read/write permission to the app, it doesn't delete files.

Although it asks for confirmation and the user accepts it

**Comment by tibbi** [comment]
does any error show up or what happens? Have you changed the default save folder?

**Comment by puchu** [comment]
yes, i changed the default save folder. No error shows up. Sometimes the files even gets removed from the player tab but the file is still in the folder and shows up on the next startup

**Comment by farline99** [comment]
Same here...

**Comment by schneidr** [comment]
Any progress on this? This is really annoying.

**Comment by tibbi** [comment]
what app and OS version does it happen to?

**Comment by schneidr** [comment]
> what app and OS version does it happen to?

App Version is 5.11.4. it happened with previous versions as well.

The device in question is a Samsung A31 with Android 12

On a Sony Xperia II with Android 12 deleting the recordings works.

Maybe a Samsung specific problem? Samsung is notorious for issues like this.

**Comment by Aga-C** [comment]
I've checked it on OnePlus with Android 13 on the latest app version - it also doesn't remove files after changing directory. App shows that the file has been deleted, but in file explorer I still can see it. The same behavior is on the emulator. 

Also, there is another related bug with removing files in a default directory. Here are the steps to reproduce:

1. Install the app from the scratch to make sure no default settings has been changed.
2. Record a file (in a default directory).
3. Change directory to another.
4. Revert to the default directory.
5. Record another file.
6. Remove both files.

It removes only the file that was originally recorded before changing the default directory. Even if I return to the default directory after changing it before, it still can't remove files in it, despite all recorded before changing anything. To make it work again, I need to revoke app's external memory permission in Android's app settings.

**Comment by schneidr** [comment]
THANK you @Aga-C . I can confirm on my Xperia that the deletion stops working as soon as I choose another directory. Reverting to the old directory does not resolve it.

I had to reset the app, now the deletion is working again. No reinstallation necessary.

**Comment by ondrejnedoma** [comment]
Any progress? Encountered this on PixelExperience android 13 on a redmi note 11, when trying to delete, a dialog with "allow voice recorder to delete audio file?" pops up, and then a huge lag, followed by the file not actually being deleted.

**Comment by RustoMCSpit** [comment]
if this bug occurs could there be a warning to say file deletion failef
